pkg/csi/controller.go

```diff
@@ -80,6 +81,20 @@ func (cs *controllerServer) isDiskShareable(volumeCapabilities []*csi.VolumeCapa
 	return false
 }
 
+func IsBlockVolume(volumeCapabilities []*csi.VolumeCapability) (bool, error) {
+	for _, volumeCapability := range volumeCapabilities {
+		if _, ok := VolumeCapabilityAccessModesStringMap[volumeCapability.AccessMode.Mode.String()]; !ok {
+			return false, fmt.Errorf("unknown volume capability [%s] or not supported", volumeCapability.String())
```

Comment:
Is this log statement identifiable to a particular Disk request? I am not sure volumeCapability.String() has any details about disk name or id.

Reply:
This is logged by the caller (and the grpc call finally)